adaptorsensor/GUI.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package adaptorsensor;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.GridLayout;
+import javax.swing.GroupLayout;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.border.TitledBorder;
+
+
+public class GUI extends JFrame{
+		public GUI(Adaptor a, Adaptor b, Adaptor c) {
+			setTitle("Sensor Tracker");
+			setLayout(new GridLayout(3,1));
+			JPanel  aPnl = new JPanel();
+			
+			aPnl.setBorder(new TitledBorder(a.getName()));
+			GridLayout aLayout = new GridLayout(0,2);
+			aPnl.setLayout(aLayout);
+			add(aPnl);
+			double aValue = a.getValue();
+			String aStatus = a.getStatus();
+			JPanel aColor = new JPanel();

```

Comment:
again variable names might be the thing to change here. however, everything else is structured quite nicely.